src/Tests/Microsoft.NET.TestFramework/TestContext.cs

```diff
@@ -45,14 +45,7 @@ public static TestContext Current
             }
         }
 
-        // For test purposes, override the implicit .NETCoreApp version for self-contained apps that to builds thare 
-        //  (1) different from the fixed framework-dependent defaults (1.0.5, 1.1.2, 2.0.0)
-        //  (2) currently available on nuget.org
-        //
-        // This allows bumping the versions before builds without causing tests to fail.
```

Comment:
We've lost this property again. Note that we will need to bump this for security updates before they are available on nuget.org or even to this repo from anywhere.

Reply:
The logic we had before felt too complex to me, and was never going to test the actual roll-forward behavior (since it always used a fake version to roll forward to when testing).  My thinking was that we could set `LatestNetCorePatchVersion` as a way to override this in tests.  Thinking about it more now, I'm not sure how well that would work.

Reply:
@nguerrera I've added properties that allow the latest patch for each minor version of .NET Core to be overridden.  We could use these to override this from tests like we were doing before, but I think a better solution would be to set the latest version in the CLI via BundledVersions.props.

Reply:
👍 Love the BundledVersions approach :)